docs/reference/createPlugin-feature-flags.md

```diff
@@ -0,0 +1,68 @@
+# createPlugin - feature flags
+
+The `featureFlags` object passed to the `register` function makes it possible for plugins to register Feature Flags in Backstage for users to opt into. You can use this to split out logic in your code for manual A/B testing, etc.
+
+```typescript
+export type FeatureFlagsHooks = {
+  registerFeatureFlag(name: FeatureFlagName): void;
+};
+```
+
+## Using with useApi
+
+This is the **recommended** way of using the API. It's officially supported by Backstage. To use it, you'll first need to register the `FeatureFlags` API via `ApiRegistry` in your `apis.ts` in your App:
```

Comment:
Awesome that you add this, and we will definitely want this to end up having a link to a central doc on how to use and include APIs in Backstage.